cmd/cmd/server.go

```diff
@@ -42,7 +42,9 @@ to quickly create a Cobra application.`,
 }
 
 func init() {
-	rootCmd.AddCommand(serverCmd)
+	// TODO: Complete functionility in server summary
```

Comment:
Let's not use TODOs to track tasks. We need to track them using Github Issues. Please create an issue and remove this comment.